docs/blog/pydantic-v2.md

```diff
@@ -0,0 +1,926 @@
+I've decided to go with the simplest and clearest approach, at the expense of a bit more typing:
+
+All methods on models will start with `model_`, fields' names will not be allowed to start with `"model"`
+(aliases can be used if required).
+
+This will mean `BaseModel` will have roughly the following signature.
+
+```{.py .annotate title="New BaseModel methods"}
+class BaseModel:
+    model_fields: List[FieldInfo]
+    """previously `__fields__`, although the format will change a lot"""
+    @classmethod
+    def model_validate(cls, data: Any, *, context=None) -> Self:  # (1)
+        """
+        previously `parse_obj()`, validate data
+        """
+    @classmethod
+    def model_validate_json(
+        cls,
+        data: str | bytes | bytearray,
+        *,
+        context=None
+    ) -> Self:
+        """
+        previously `parse_raw(..., content_type='application/json')`
+        validate data from JSON
+        """
+    @classmethod
+    def model_is_instance(cls, data: Any, *, context=None) -> bool: # (2)
+        """
+        new, check if data is value for the model
+        """
+    @classmethod
+    def model_is_instance_json(
+        cls,
+        data: str | bytes | bytearray,
+        *,
+        context=None
+    ) -> bool:
+        """
+        Same as `model_is_instance`, but from JSON
+        """
+    def model_dump(
+        self,
+        include: ... = None,
+        exclude: ... = None,
+        by_alias: bool = False,
+        exclude_unset: bool = False,
+        exclude_defaults: bool = False,
+        exclude_none: bool = False,
+        mode: Literal['unchanged', 'dicts', 'json-compliant'] = 'unchanged',
+        converter: Callable[[Any], Any] | None = None
+    ) -> Any:
+        """
+        previously `dict()`, as before
+        with new `mode` argument
+        """
+    def model_dump_json(self, ...) -> str:
+        """
+        previously `json()`, argument as above
+        effectively an alias of `json.dump(self.model_dump(..., mode='json'))`
+        """
+    def model_json_schema(self, ...) -> dict[str, Any]:
+        """
+        previously `schema()`, arguments roughly as before
+        JSON schema as a dict
+        """
+    def model_update_forward_refs(self) -> None:
+        """
+        previously `update_forward_refs()`, update forward references
+        """
+    def model_copy(self, ...) -> Self:
+        """
+        previously `copy()`, arguments roughly as before
+        copy a model
+        """
+    @classmethod
+    def model_construct(
+        self,
+        _fields_set: set[str] | None = None,
+        **values: Any
+    ) -> Self:
+        """
+        previously `construct()`, arguments roughly as before
+        construct a model with no validation
+        """
+    @classmethod
+    def model_customise_schema(cls, schema: dict[str, Any]) -> dict[str, Any]:
```

Comment:
I'm sorry for my British friends but since we don't write colour and behaviour I guess we should write customize haha

Reply:
🙈 ok...